From 560ff4201ceec40e5b9921b6d1baacfba656e42a Mon Sep 17 00:00:00 2001
From: Cuong Nguyen <128072568+can-anyscale@users.noreply.github.com>
Date: Mon, 8 Jan 2024 08:07:59 -0800
Subject: [PATCH] [ci] remove boost as a dependency (#42226)

The current boost rule is busted in master branch: https://buildkite.com/ray-project/postmerge/builds/2404#018ce49f-f8c1-4518-9d32-868a5a4decc6/176-281. Historically, we need to separate the boost rule from com_github_nelhage_rules_boost since we need a patch. This is no longer the case since #32550.

Signed-off-by: can <can@anyscale.com>
---
 bazel/ray_deps_setup.bzl | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/bazel/ray_deps_setup.bzl b/bazel/ray_deps_setup.bzl
index c406ba9281..f91eb303c4 100644
--- a/bazel/ray_deps_setup.bzl
+++ b/bazel/ray_deps_setup.bzl
@@ -159,15 +159,6 @@ def ray_deps_setup():
         sha256 = "97e70364e9249702246c0e9444bccdc4b847bed1eb03c5a3ece4f83dfe6abc44",
     )
 
-    auto_http_archive(
-        # This rule is used by @com_github_nelhage_rules_boost and
-        # declaring it here allows us to avoid patching the latter.
-        name = "boost",
-        build_file = "@com_github_nelhage_rules_boost//:BUILD.boost",
-        sha256 = "71feeed900fbccca04a3b4f2f84a7c217186f28a940ed8b7ed4725986baf99fa",
-        url = "https://boostorg.jfrog.io/artifactory/main/release/1.81.0/source/boost_1_81_0.tar.bz2",
-    )
-
     auto_http_archive(
         name = "com_github_nelhage_rules_boost",
         # If you update the Boost version, remember to update the 'boost' rule.
-- 
2.34.1

